src/js/actions/guides.js

```diff
@@ -320,6 +319,10 @@ define(function (require, exports) {
      * @return {Promise}
      */
     var queryCurrentGuides = function (document) {
+        var appStore = this.flux.store("application");
+
+        document = document || appStore.getCurrentDocument();
```

Comment:
Need a JSDoc fix up if this param is optional. Also, it would be better to do this the right way, testing for `document === undefined`.